packages/@aws-cdk/aws-codepipeline-actions/lib/private/cloudformation/_singleton-policy.ts

```diff
@@ -0,0 +1,172 @@
+import * as iam from '@aws-cdk/aws-iam';
```

Comment:
I commented on this file here: https://github.com/aws/aws-cdk/pull/14225#discussion_r802150136.